game/views.py

```
@@ -429,31 +428,27 @@ def offer_draw(request):
-
     game_status = 'resignation'
 
     game.game_status = game_status
     request.session['game'] = game.to_dict()
     request.session.modified = True
 
-    record_game_result(request, game.mode, winner, 'resign', game.player_color)
+    try:
+        record_game_result(request, game.mode, winner, 'resign', game.player_color)
+    except Exception:
+        pass  # lets check this !
```

Ruff (0.15.13)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)